<?xml version="1.0"?>
<launch>
    <!--Launch Zed Wrapper-->
	<include file="$(find zed_wrapper)/launch/zed2.launch"/>

    <!--Down sampling-->
    <node pkg="sensor_fusion" type="camera_downsample" name="camera_downsample">
        <param name="ds_size" type="double" value="0.05" />
        <remap from="/cloud" to="/zed2/point_cloud/cloud_registered" />
        <remap from="/output" to="/zed2/ds_cloud" />
    </node>
</launch>
